tests/Domain/Model/ValueObject/CollectionValueObjectTest.php

```diff
@@ -53,8 +53,11 @@ public function given_collection_when_ask_to_sort_then_return_new_sorted_collect
     {
         $collection = CollectionValueObject::from([5, 1, 4, 2, 3]);
         $sorted = $collection->sort(
-            function ($a , $b){
-                return $a > $b;
+            function ($a, $b) {
+                if ($a == $b) {
+                    return 0;
+                }
+                return ($a < $b) ? -1 : 1;
```

Comment:
Esa lógica podemos hacer la mas compacta y con menos comprobaciones de la siguiente manera:

```suggestion
               return $a - $b;
```